mongoose_dashboard/app.js

```diff
@@ -0,0 +1,90 @@
+var express = require('express');
+var app = express();
+var bodyParser = require('body-parser');
+var path = require('path');
+var mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/mongo_dashboard');
+
+var MongooseSchema = new mongoose.Schema({
+	name: {type: String}
+}, {timestamps: true});
+
+mongoose.model('Mongoose', MongooseSchema);
+
+var Mongoose = mongoose.model('Mongoose');
+
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(express.static(path.join(__dirname, "./static")));
+
+app.set('views', path.join(__dirname, "./views"));
+app.set('view engine', 'ejs');
+
+app.get('/', function(request, response){
+	Mongoose.find({}, function(err, mongooses){
+		if (err){
+			console.log('Errors');
```

Comment:
Always console.log your err object we get from the callback, and consider how you would send these error messages to the client!